CHANGELOG.md

```diff
@@ -38,6 +38,14 @@ A brief description of the categories of changes:
 ### Added
 * (py_wheel) Now supports `compress = (True|False)` to allow disabling
   compression to speed up development.
+* (toolchains) New Python versions available: 3.13.0rc2 using the [20240909] release.
+* (toolchains): Bump default toolchain versions to:
+    * `3.8 -> 3.8.20`
+    * `3.9 -> 3.9.20`
+    * `3.10 -> 3.10.15`
+    * `3.11 -> 3.11.10`
+    * `3.12 -> 3.12.6`
```

Comment:
I think this part of the change is lost somewhere in the `git` dungeons. Could you please check if you have pushed it?